pkg/Library/NodeGraph/ConnectionUpdator.js

```diff
@@ -72,28 +72,43 @@ updatePipelineConnections(pipeline, nodeTypes, candidates) {
 },
 
 updateNodeConnections(node, nodeType, candidates) {
-  if (this.shouldUpdateConnections(node) && candidates) {
+  if (candidates) {
+    let changed = this.initializeConnections(node, nodeType, candidates);
+    changed |= this.updateNoDisplayConnections(node, nodeType, candidates);
```

Comment:
Boolean OR? Do you mean "||="?

Reply:
ok that was a confusing statement, rewrote for better readability.